changelog/unreleased/enhancement-indicate-shares-not-managable-due-to-file-locking

```diff
@@ -0,0 +1,7 @@
+Enhancement: Indicate shares that are not manageable due to file locking
```

Comment:
```suggestion
Bugfix: Indicate shares that are not manageable due to file locking
```
... since it's a UI bugfix IMO 🙈

Reply:
We could argue about this, since this new behaviour was just brought in by OCIS.... but i opt out by just change it

Reply:
Ah, wasn't aware... sorry.  then enhancement is fine of course!

Reply:
No problem, I don't think we need to change it over again (: